Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -411,20 +411,24 @@ extension KeyboardShortcuts {
 		AsyncStream { continuation in
 			let id = UUID()
 
-			streamKeyDownHandlers[name, default: [:]][id] = {
-				continuation.yield(.keyDown)
-			}
-			streamKeyUpHandlers[name, default: [:]][id] = {
-				continuation.yield(.keyUp)
-			}
+			DispatchQueue.main.async {
```

Comment:
Would be better to use `@MainActor`.

Reply:
I tried using `@MainActor` but I got a bunch or errors saying `Static property 'streamKeyDownHandlers' isolated to global actor 'MainActor' can not be mutated from a non-isolated context`.

To be honest, I don't understand actors well enough to know how to implement it that way.

Reply:
The error is protecting you from a possible race condition.

But yes, probably not worth caring about right now.